lib/js/components/Buttons/Button/Button.stories.ts

```diff
@@ -0,0 +1,97 @@
+import Button from './Button.vue';
+import {
+	BUTTON_SIZES,
+	BUTTON_TYPES,
+	BUTTON_COLORS,
+	BUTTON_STATES,
+	BUTTON_RADIUSES,
+	BUTTON_ELEVATIONS,
+} from './Button.consts';
+import { ICONS } from '../../Icon';
+
+import { Meta, StoryFn } from '@storybook/vue';
+
+export default {
+	title: 'Components/Buttons/Button',
+	component: Button,
+} as Meta<typeof Button>;
+
+const StoryTemplate: StoryFn<{
+	slotText: Object;
```

Comment:
I'm wondering if it's ok that we pass obejct with props as a type. In docs I see different approach: https://storybook.js.org/docs/vue/writing-stories/introduction#using-args

Reply:
Hmm it wasn't simple, I need to define `Args` and `ArgTypes` types of variables, @kvas-damian check code. But it works fine.

Reply:
@kvas-damian my God, it looks nice but throwed me back to the problem:
![Screenshot 2022-09-20 at 15 33 28](https://user-images.githubusercontent.com/4208462/191764911-29ab1d59-9b41-4b28-ae7e-53990fca704a.png)

Reply:
is it related to the change of return type in `StoryFn`?